services/FeefoReviewService.php

```diff
@@ -30,9 +30,9 @@ public function feefoMerchantId() {
   private function getProducts() {
     $cacheKey = 'feefo_'.$this->feefoMerchantId();
     $products = craft()->cache->get($cacheKey);
-    if (is_null($products)) {
+    if ($products === false) {
       $products = $this->getProductReviews();
-      if (!is_null($products)
+      if ($products != null)
```

Comment:
we should still use the !is_null function here

Reply:
Change it!